mteb/leaderboard/utils.py

```diff
@@ -0,0 +1,64 @@
+import json
+from pathlib import Path
+from typing import Literal
+
+import mteb
+
+# Model sizes in million parameters
+MIN_MODEL_SIZE, MAX_MODEL_SIZE = 0, 100_000
```

Comment:
These seems like leaderboard configs. I would probably move to config.py (potentially together with main leaderboard etc.)